lib/still/preprocessor/output_path.ex

```diff
@@ -21,7 +21,9 @@ defmodule Still.Preprocessor.OutputPath do
       when not is_nil(extension) do
     output_file =
       input_file
-      |> String.replace(Path.extname(input_file), extension)
+      |> String.split(".")
+      |> hd()
+      |> Kernel.<>(extension)
```

Comment:
Does this support multiple processing of different extensions? So `.eex.html` - are we able to process EEx first and then HTML?

Reply:
No no, file names have nothing to do with preprocessors. The issue is that if I name a file `index.html.eex` for some reason, the output file would be `index.html.html`. We only replace the last extensions